src/components/FormItem.vue

```diff
@@ -11,22 +11,33 @@
                 v-bind="getCommonProperties()"
             />
 
+            <v-text-field
+                v-else-if="field.widget.type === 'password'"
+                :type="show_password ? 'text' : 'password'"
+                :append-icon="show_password ? 'visibility' : 'visibility_off'"
+                v-bind="getCommonProperties()"
+                @click:append="show_password = !show_password"
```

Comment:
It seems that there is one universal `show_password` variable, that affects all password fields on the page. It would be better if we could show or hide each password individually.

Reply:
No, becuase each FormIte should have only one field. And form is containg more FormItems - so showing password of each password should be individual.
But I can create PasswordField component and separete it

Reply:
Ah, I see. I just overlooked that we are inside `FormItem` component. My mistake.